public/data/food/ingredients.json

```diff
@@ -1,566 +1,839 @@
 [
   {
     "id": "egg",
-    "name": "Oeuf",
-    "category": "animal_product",
+    "name": "Egg, national average, at farm gate {FR} U",
+    "categories": [
+      "animal_product"
+    ],
     "default": "AGRIBALU000000003105162",
     "default_origin": "EuropeAndMaghreb",
     "raw_to_cooked_ratio": 0.974,
     "density": 1,
     "inedible_part": 0.2,
     "transport_cooling": "none",
     "visible": true,
-    "variants": {
-      "organic": {
-        "process": "AGRIBALU000000003105163",
-        "beyondLCA": {
-          "agro-diversity": 50,
-          "agro-ecology": 50,
-          "animal-welfare": 50
-        }
-      },
-      "bleu_blanc_coeur": "Egg Bleu Blanc Coeur outdoor system at farm gate"
-    }
+    "agro-diversity": 10,
+    "agro-ecology": 10,
+    "animal-welfare": 10
```

Comment:
The desired format is 

```elm
  "complements": {
    "agro-diversity": 10,
    "agro-ecology": 10,
    "animal-welfare": 10
  }
```

Reply:
Can't we flatten a little more and remove complements/beyondLCA ?

Reply:
I'm against it as regrouping things is a hobby of mine, but if this represents a technical limitation slowing you down, I'm fine with hard flatting